public/app/features/dashboard/api/ResponseTransformers.test.ts

```diff
@@ -1,19 +1,52 @@
 import { DashboardV2Spec } from '@grafana/schema/dist/esm/schema/dashboard/v2alpha0/dashboard.gen';
-import { DashboardDTO } from 'app/types';
+import {
+  AnnoKeyCreatedBy,
+  AnnoKeyDashboardId,
+  AnnoKeyFolder,
+  AnnoKeySlug,
+  AnnoKeyUpdatedBy,
+  AnnoKeyUpdatedTimestamp,
+} from 'app/features/apiserver/types';
+import { DashboardDataDTO, DashboardDTO } from 'app/types';
 
 import { ResponseTransformers } from './ResponseTransformers';
 import { DashboardWithAccessInfo } from './types';
 
 describe('ResponseTransformers', () => {
   describe('v1 transformation', () => {
     it('should transform DashboardDTO to DashboardWithAccessInfo<DashboardV2Spec>', () => {
-      const dashboardDTO: DashboardDTO = {
-        meta: {
-          created: '2023-01-01T00:00:00Z',
-          createdBy: 'user1',
-          updated: '2023-01-02T00:00:00Z',
-          updatedBy: 'user2',
-          folderUid: 'folder1',
+      const dashboardV1: DashboardDataDTO = {
+        uid: 'dashboard-uid',
+        id: 123,
+        title: 'Dashboard Title',
+        description: 'Dashboard Description',
+        tags: ['tag1', 'tag2'],
+        schemaVersion: 1,
+        graphTooltip: 0,
+        preload: true,
+        liveNow: false,
+        editable: true,
+        time: { from: 'now-6h', to: 'now' },
+        timezone: 'browser',
+        refresh: '5m',
+        timepicker: {
+          refresh_intervals: ['5s', '10s', '30s'],
+          hidden: false,
+          time_options: ['5m', '15m', '1h'],
+          nowDelay: '1m',
+        },
+        fiscalYearStartMonth: 1,
+        weekStart: 'monday',
+        version: 1,
+        links: [],
```

Comment:
Let's add some links to ensure we transform them properly since we already support the in the transformers

Reply:
added 84e3e124fdd78502e5d069437e30b56b0269deef :)